src/mapeditdialog.cpp

```diff
@@ -1,100 +1,101 @@
-#include <QDebug>
 #include <QFileDialog>
 #include <QMessageBox>
-#include <QString>
 
+#include "editor/map.hpp"
 #include "mapeditdialog.hpp"
 #include "ui_mapeditdialog.h"
 
-#include "editor/map.hpp"
-
 MapEditDialog::MapEditDialog(
     const std::shared_ptr<const Editor::Project> project,
     std::shared_ptr<Misc::MapDocument> mapDocument,
-    QWidget *parent) : QDialog(parent),
-                       ui(new Ui::MapEditDialog),
-                       m_project(project)
+    QWidget *parent)
+    : QDialog(parent)
+    , m_ui(new Ui::MapEditDialog)
+    , m_project(project)
 {
```

Comment:
This is the reason to use the ":" or the "," as the first element of the line for constructor initialization list : 

When the constructor arguments are splitted on several lines, having the syntax at the begining of the line help to see at first glance which line corresponds to a parameter and which to an initialization.